firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/ImagenGenerationConfig.kt

```diff
@@ -1,19 +1,64 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package com.google.firebase.vertexai.type
 
+/**
+ * Contains extra settings to configure image generation.
+ *
+ * @param negativePrompt This string contains things that should be explicitly excluded from
+ * generated images.
+ * @param numberOfImages How many images should be generated.
+ * @param aspectRatio The aspect ratio of the generated images.
+ * @param imageFormat The file format/compression of the generated images.
+ * @param addWatermark Adds an invisible watermark to mark the image as AI generated.
+ */
 public class ImagenGenerationConfig(
   public val negativePrompt: String? = null,
   public val numberOfImages: Int? = 1,
   public val aspectRatio: ImagenAspectRatio? = null,
   public val imageFormat: ImagenImageFormat? = null,
   public val addWatermark: Boolean? = null,
 ) {
+  /**
+   * Builder for creating a [ImagenGenerationConfig].
+   *
+   * Mainly intended for Java interop. Kotlin consumers should use [imagenGenerationConfig] for a
```

Comment:
Do you think that this is visible to the devs in the generated ref docs?

Reply:
Yes, its visible for GenerationConfig

Reply:
https://firebase.google.com/docs/reference/kotlin/com/google/firebase/vertexai/type/GenerationConfig.Builder

Reply:
added a suggested edit :-)